docs/index.md

```diff
@@ -65,7 +65,8 @@ resource "metal_device" "web1" {
 
 The following arguments are supported:
 
-* `auth_token` - (Required) This is your Equinix Metal API Auth token. This can also be specified
-  with the `PACKET_AUTH_TOKEN` shell environment variable.
+* `auth_token` - (Required) This is your Equinix Metal API Auth token. This can
```

Comment:
maybe don't mention PACKET_AUTH_TOKEN since we deprecate it

Reply:
I'll still mention it because I don't want people to be surprised when things still work because they forgot they have `PACKET_AUTH_TOKEN` set. I'll mention that it is deprecated.

Reply:
968aad2 includes this message